build.gradle

```diff
@@ -118,6 +118,7 @@ subprojects {
             // JDepend manages to include a different version of itself in its own transitive closure.
             force "jdepend:jdepend:${project.jdepend.toolVersion}"
         }
+        resolutionStrategy.cacheChangingModulesFor 0, 'seconds'
```

Comment:
OK with me.  Pretty soon we'll need to start having app-tools-lib version
their releases.

On Wed, Jun 8, 2016 at 3:23 PM, Etan Shaul <notifications@github.com> wrote:

> In build.gradle
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/700#discussion_r66322008>
> :
>
> > @@ -118,6 +118,7 @@ subprojects {
> >              // JDepend manages to include a different version of itself in its own transitive closure.
> >              force "jdepend:jdepend:${project.jdepend.toolVersion}"
> >          }
> > +        resolutionStrategy.cacheChangingModulesFor 0, 'seconds'
>
> @chanseokoh <https://github.com/chanseokoh> The problem is that Travis
> won't let us merge it.
>
> We have an annoying chicken and egg problem here though: if I move this to
> a separate PR, that PR will fail (because it would not have been sync'd up
> with the common-lib) - if I sync it, it will basically be throw away since
> all those classes have been refactored by this PR. If you all are ok with
> it this time, I'll leave the common-lib syncing and cache update all in
> this PR. Next time I can be more careful with the timing.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/700/files/a3a2318d273b9729e13906620a2014cc69c7e449#r66322008>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AHf5HWemacwuebmmgP6IBGuaTPyYc-Byks5qJxaegaJpZM4Im4aR>
> .
>